kiss_slam/kiss_slam_pybind/occupancy_mapper/occupancy_mapper.hpp

```diff
@@ -40,6 +40,7 @@ class OccupancyMapper {
 
     void IntegrateFrame(const Vector3fVector &pointcloud, const Eigen::Matrix4f &pose);
     std::tuple<Vector3iVector, std::vector<float>> GetOccupancyInformation() const;
+    Vector3iVector GetOccupiedVoxels(float probability_threshold) const;
```

Comment:
1. The method argument should be marked const.
 2. It could also be renamed to `occupancy_probability_threshold`  to make it more explicit.